### DISCUSSION TITLE
How to enable ServiceMonitor not using helm?

### DISCUSSION THREAD

**Discussion by luckercs** [milvus-io/milvus#26753] [Q&A and General discussion]
We install Milvus Cluster with Milvus Operator.

When Deploying Monitoring Services on Kubernetes,  we not find "xxx.yaml" file to  enable ServiceMonitor on ["https://milvus.io/docs/monitor.md".](https://milvus.io/docs/monitor.md)

How to enable ServiceMonitor not using helm? Or How to deploy monitoring services on Kubernetes not using helm?

**Comment by xiaofan-luan** [comment]
@zwd1208 @haorenfsa 
any ideas

**Reply by haorenfsa** [reply]
@luckercs Hi, Milvus Operator uses PodMonitor instead of ServiceMonitor to monitor milvus pods (since not all milvus pods have service to export). PodMonitor is added by default by Milvus Operator if you have already installed prometheus operator in your kubernetes.
